### ISSUE TITLE
Running for whole genome sequencing (+ large VCF)

### ISSUE THREAD

**Issue by NikdAK** [rvalieris/signeR#19]
Hi,
I am currently trying to run your pipeline for my human whole genome sequencing data.
Regarding this I have a few questions:

1) I assume your opportunity matrix is the background model. Apparently you normally expect the bed file to be the bait regions in targeted sequencing.
Should I use the complete chromosome boundaries as bed file? e.g.  chr1	1	248956422
Or alternatively a certain window before/after each mutation?

2) For some samples I do not have control tissue and therefore very large VCF files (~200MB) due to germline mutations.
genCountMatrixFromVcf() is having issues loading large data like this, i.e. loading forever. Is there are way to bypass this?
Filtering by allele frequency is not an option, since I am also interested in low frequency variants.

3) I did a test run on my small samples without the opportunity matrix and got the correct plot. Anyways for some reason the field headers are missing (C>A, ...). How can that be? They appear if I try to use your example data.
![signeR_screen](https://user-images.githubusercontent.com/39265538/101772623-e47c2980-3aeb-11eb-96be-1b01b50915f6.png)
--> EDIT: solved this one. The headers will only be shown with 3+ samples. For some reason they are not shown for 2 samples only.

Thank you very much!

**Comment by rvalieris** [comment]
Hello,

1. did you called variants for the entire chromosome ? some parts of the chromosome are hard to map or are completely missing like the centromeres and telomeres, I recommend calculating genome-wide coverage of all your samples and only using regions with uniform coverage across all.

2. for large VCFs you can split the file and parallelize the process, see this example:
https://gist.github.com/rvalieris/e8dc4c40068a6593f1f4bb19e41ad50c

**Comment by rvalieris** [comment]
another thing, if you want to measure **somatic** signatures you need to remove these germline mutations somehow. Ideally with paired normals, but even if you don't have those you can still:
* filter >1% mutations in a population of dbSNP or gnomAD databases
* pool all unmatched normals of your cohort together and use it to filter all tumor samples

**Comment by NikdAK** [comment]
Thank you very much for this answer, it helped at lot. 

1) I did so and wrote a small script to run mosdepth across my samples to get the genome wide coverage. I then combined the regions and filtered it down to a uniform coverage (i.e. for each region -+2 median absolute deviation from the genome wide median). 
This seems to work very well. I can share the code if someone also needs this. 

2) This works, even though one has to be careful with the memory usage in R.

3) Yes I am aware of this problem. For now I already did some hard cutoffs on coverage and frequency as well as filtering based on dbSNP+gnomAD
